### ISSUE TITLE
Dependency problem on react-native@"^.0.66.4"

### ISSUE THREAD

**Issue by Goostavo** [talut/react-native-pin-view#58]
**Describe the bug**
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: qualityhub@1.2.3
npm ERR! Found: react-native@0.66.4
npm ERR! node_modules/react-native
npm ERR!   react-native@"^0.66.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-native@"^0.63.4" from react-native-pin-view@3.0.2
npm ERR! node_modules/react-native-pin-view
npm ERR!   react-native-pin-view@"^3.0.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**To Reproduce**
1. npx react-native init AwesomeProject
2. npm install --save react-native-pin-view

**Expected behavior**
Updates dependency

**Comment by StackOverValdex** [comment]
i have this bug installing the latest version :'-(

**Comment by anisamirouche** [comment]
is there any solution ?

**Comment by DianaKoenraadt** [comment]
Same problem here, except now with  "react-native": "0.70.5",

**Comment by DianaKoenraadt** [comment]
@talut Are you still developing this or have you abandoned it?

**Comment by talut** [comment]
Hey @DianaKoenraadt  I'm so full with my daily job but I'm gonna try to update all my libraries. So this fixed with v3.0.3 Please check new version.

**Comment by DianaKoenraadt** [comment]
@talut  I understand. It would have been fine if you had said you didn't support it anymore, then at least there would be clarity. But even better of course is that you're willing to update the library, great!
